packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping-anchor.md

```diff
@@ -0,0 +1,197 @@
+
+### Sizing
+
+The column should support the same sizing modes as the text column, which is fractional widths plus minimum pixel widths.
+
+### Keyboard Interactions
+
+Arrowing to a anchor table cell should focus the link (if it is actually a link and not just a text span). Pressing `Enter` on a focused link will trigger navigation.
+
+### Accessibility Roles
+
+In the accessibility tree, the cells of an anchor column are instances of [`nimble-table-cell`](https://github.com/ni/nimble/blob/f663c38741e731bef91aa58e8fb2d1cec653b679/packages/nimble-components/src/table/components/cell/template.ts#L6) which has a `role` of [`cell`](https://w3c.github.io/aria/#cell). The cell then has a child `nimble-anchor`, which has a `role` of [`link`](https://w3c.github.io/aria/#link).
+
+### Angular integration
+
+Angular directives will be created for the column component. No component has form association, so a `ControlValueAccessor` will not be created.
+
+#### Angular Routing support
+
+Using the `TableColumnAnchor` in Angular requires the use of the [`NavigationGuard`](https://github.com/ni/nimble/blob/main/packages/angular-workspace/nimble-angular/table-column/anchor/nimble-table-column-anchor-navigation-guard.directive.ts) in order to allow proper routing within the Angular application.
+
+### Blazor integration
+
+Blazor wrappers will be created for the components. Columns will be generic in the type of the key, and will cascade that type parameter to contained mapping elements (see [`CascadingTypeParameter`](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/generic-type-support?view=aspnetcore-7.0#cascaded-generic-type-support)):
+
+```HTML
+<NimbleTableColumnMappingAnchor TKey=string Field="Status" LabelField="StatusField" HrefField="StatusURL">
+    <NimbleMappingIcon
+        Key="@("success")"
+        Icon="nimble-icon-check"
+        Severity="IconSeverity.Success">
+    </NimbleMappingIcon>
+    <NimbleMappingSpinner
+        Key=@("calculating")>
+    </NimbleMappingSpinner>
+</NimbleTableColumnMapping>
+```
+
+### Globalization
+
+All text will be provided by the client and is expected to be localized.
+
+### Security
+
+N/A
+
+### Performance
+
+N/A
+
+### Dependencies
+
+None
+
+### Test Plan
+
+- Unit tests will be written verifying the usual component expectations, plus:
+    - renders mapping matching the cell value (string, number, and boolean)
+    - nothing rendered when value matches no mappings
+    - validation error when non-unique mapping keys exist
+    - validation error when invalid icon name given
+    - grouping header for icon column includes text
+- Verify manually that the column content appears in the accessibility tree and can be read by a screen reader.
+- Verify manually that several mapping columns with thousands of elements scrolls performantly.
+- Visual Chromatic tests will be created
+
+### Tooling
+
+N/A
+
+### Documentation
+
+Documented in Storybook
+
+---
+
+## Open Issues
```

Comment:
Depending on which way we go on visual design, if we end up not wanting to show icon text for the icon mappings I propose we enforce that with validation. Require in mapping validation that all the icon mappings have "text-hidden" true (as opposed to ignoring the attribute on the mapping or creating a new icon mapping that does not have the unused attribute).